project-koku/koku
# Add a migration to wipe data from org units table

### Description
Recently in #cost-mgmt-alerts we have seen the following error: 
```
AWSOrganizationalUnit.MultipleObjectsReturned
get() returned more than one AWSOrganizationalUnit -- it returned 2!
environment
prod
```
We think this is because of the recent migration to OSD4 & impeccable timing caused both clusters to create an entry for the same org unit in the db. We decided that the best course of action was to clear out the org units (and refs) from the db and start fresh since the crawler will run each day. This PR adds a migration for a one-time removal of the org units and their references from the cost reports. 


### Testing
1. On master run `make load-aws-org-unit-tree`
2. Connect to the db and view the organizational unit table (and optionally the aws cost summary table): 
```
psql postgres -U postgres -p 15432 -h localhost
postgres=# SELECT * FROM acct10001.reporting_awsorganizationalunit;  
postgres=# SELECT * FROM acct10001.reporting_awscostentrylineitem_daily_summary;
```
3. Checkout this branch and run `make run-migrations` 
4. View the above tables in the database
5. Run `make load-aws-org-unit-tree` & view the tables/make sure it works as normal
